src/test/java/ParserTest.java

```diff
@@ -0,0 +1,16 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ParserTest {
+    @Test
+    public void hasNoArgs_reallyHasNoArgs_returnsTrue() {
```

Comment:
LOL at your function name here. but instead of `reallyHasNoArgs`, call it `noArgs` so that its more succinct.